dashboard/apps/prototype/context_processors.py

```diff
@@ -0,0 +1,11 @@
+def globals(request):
```

Comment:
might be better to rename the function, as `globals` is a built-in function as well.